public/app/features/dashboard-scene/scene/FeedbackButtonNewLayout.tsx

```diff
@@ -0,0 +1,41 @@
+import { css } from '@emotion/css';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { t } from '@grafana/i18n';
+import { useStyles2 } from '@grafana/ui';
+
+import feedbackImage from '../../../../img/icons/unicons/feedback.svg';
+
+export function FeedbackButtonNewLayout() {
+  const styles = useStyles2(getStyles);
+  return (
+    <a
+      href="https://docs.google.com/forms/d/e/1FAIpQLSfDZJM_VlZgRHDx8UPtLWbd9bIBPRxoA28qynTHEYniyPXO6Q/viewform"
```

Comment:
One thing that I'm thinking: don't hardcode the URL here. I think we can use the `grafana.ini` config to pass the URL. If the URL is not set, we don't show this. This way we can control if this is shown Cloud-only. If not, we put the URL in `grafana.ini` by default. If not, the default would be empty.